server/views/templates/explore-landing.njk

```diff
@@ -1,6 +1,8 @@
 {% extends "pages/explore-landing.njk" %}
 
 {% block body %}
+  {% component 'page-description', { title: 'Explore landing', modifiers: ['hidden'] } %}
```

Comment:
I'm not going to pretend I know how SEO works. I don't think anyone really does. But as I understand it, you're penalised if you don't have an h1 on the page and/or if your heading level structure isn't correct, so I believe this can be seen as good practice.